src/FlaUI.Core/Application.cs

```diff
@@ -13,7 +13,7 @@ namespace FlaUI.Core
 {
     public class Application : IDisposable
     {
-        private static readonly ILogger Log = new ConsoleLogger();
+        public ILogger Log { get; }

```

Comment:
Yes, it is actually used. As for it only affecting messages logged from the application class this is true in so far as the Application class is the only thing that logs at the moment. If more classes were to start logging stuff you would have to thread the application logger through the various constructors.

This is probably overkill. I'm fine removing it all together and just using the default logger everywhere.